pages/docs/couple-your-code/couple-your-code-mesh-and-data-access.md

```diff
@@ -33,8 +33,18 @@ void Participant::writeData(
     precice::span<const double>   values)
 ```
 
-<!-- TODO Also point to section where `relativeReadTime` is explained? We will probably solve this in https://github.com/precice/precice.github.io/pull/257 -->
-Similarly, there is a `readData` API function for reading coupling data.
+Similarly, there is a `readData` API function for reading coupling data:
+
+```cpp
+void readData(
+    precice::string_view          meshName,
+    precice::string_view          dataName,
+    precice::span<const VertexID> vertices,
+    double                        relativeReadTime,
+    precice::span<double>         values) const;
+```
+
+We will talk about the additional argument `relativeReadTime` in detail in [the section on time interpolation](couple-your-code-waveform.html).
```

Comment:
At least a minimum amount of information would be useful here. Otherwise, we force the reader to switch pages in order to get an idea what the argument of this fundamental function even means.

This way we give a basic explanation and a hint on where to find more information later, which shouldn't interrupt the flow of reading the page.

```suggestion
The relative read time can be anthing from the current point in time `0` and the end of the time window `getMaxTimeStepSize()`. We will talk about the additional argument `relativeReadTime` in detail in [the section on time interpolation](couple-your-code-waveform.html).
```